hazelcast/test/src/cp_test.cpp

```diff
@@ -44,6 +44,9 @@ class cp_test : public ClientTest
 
     virtual void SetUp()
     {
+        if (client_) {
+            client_->shutdown().get();
+        }
```

Comment:
Latest instance will be left. This needs to be moved to `TearDown` method.

Reply:
Fixed